packages/react-mutation-mapper/src/util/MyCancerGenomeUtils.ts

```diff
@@ -0,0 +1,110 @@
+import { IMyCancerGenome, IMyCancerGenomeData } from '../model/MyCancerGenome';
+import { Mutation } from '../model/Mutation';
+
+import myCancerGenomeJson from '../resources/mycancergenome.json';
+
+export function getMyCancerGenomeLinks(
```

Comment:
Seems like this is the only method to be tested?

Reply:
Ideally we should test all the methods, but I haven't added any new tests, just moved existing MyCancerGenome related functions (without modifying) into this new file.